app/javascript/dashboard/components/widgets/Label.vue

```diff
@@ -1,12 +1,9 @@
 <template>
-  <div
-    :class="labelClass"
-    :style="{ background: bgColor, color: textColor }"
-    :title="description"
-  >
+  <div :class="labelClass" :style="labelStyle" :title="description">
+    <i v-if="icon" class="label--icon" :class="icon" @click="onClick" />
     <span v-if="!href">{{ title }}</span>
-    <a v-else :href="href" :style="{ color: textColor }">{{ title }}</a>
-    <i v-if="showIcon" class="label--icon" :class="icon" @click="onClick" />
```

Comment:
I guess this should be kept, if we want to use `+` instead of `X`

Reply:
I have covered that with a new `<i` tag above.